spec/game_setup_spec.rb

```diff
@@ -42,19 +42,25 @@
 
   describe 'get_token' do
     it 'returns a letter token' do
-      user_interface.set_input(['a'])
-      expect(game_setup.get_token(config, setup_validator, 'Player 1')).to eql('a')
+      user_interface.set_input(['150'])
+      expect(game_setup.get_token(config, setup_validator, 'Player 1')).to eql("\u{1f4a9}")
     end
 
     it 'tries again if token is not valid' do
-      user_interface.set_input(['1', 'a'])
-      expect(game_setup.get_token(config, setup_validator, 'Player 1')).to eql('a')
+      user_interface.set_input(['a', '150'])
+      expect(game_setup.get_token(config, setup_validator, 'Player 1')).to eql("\u{1f4a9}")
     end
 
     it 'tries again if token is taken' do
-      user_interface.set_input(['a', 'b'])
-      game_setup.instance_variable_set(:@token1, 'a')
-      expect(game_setup.get_token(config, setup_validator, 'Player 1')).to eql('b')
+      user_interface.set_input(['150', '149'])
+      game_setup.instance_variable_set(:@token1, '150')
```

Comment:
😢

Reply:
I think this test is a little outdated with some of the other changes.  I've added a [method](https://github.com/saralein/tic-tac-toe_ruby/pull/22/files#diff-b1e3c91fecf2e775c58f0ba6d8acb4ecR77) to remove token 1 from emojis and included a [test](https://github.com/saralein/tic-tac-toe_ruby/pull/22/files#diff-affc662720a70899519e4bd6f5bd03e0R61) to make sure this happens.  This way, they will never have the option to pick token 1, only to pick something not in range.

I'm pretty sure you were saddened by instance_variable set.  I've updated another game setup test to [remove that](https://github.com/saralein/tic-tac-toe_ruby/pull/22/files#diff-affc662720a70899519e4bd6f5bd03e0L94).